src/lib/stitching/mergeSchemas.ts

```diff
@@ -56,3 +59,39 @@ export const mergeSchemas = async () => {
 
   return mergedSchema
 }
+
+// The end goal:
```

Comment:
do you want to keep these commented code around?

Reply:
I do, I'm OK with this for now, I'll remove it once we hit some kind of partiy